lib/index.ts

```diff
@@ -93,19 +106,20 @@ export async function inspect(
   const mvnArgs = buildArgs(targetFile, options.args);
   const mavenCommand = getCommand(root, targetFile);
   const targetFilePath = calculateTargetFilePath(
-    mavenCommand,
+    mavenCommand.command,
     root,
     targetPath,
   );
+
   try {
-    const result = await subProcess.execute(mavenCommand, mvnArgs, {
-      cwd: targetFilePath,
+    const result = await subProcess.execute(mavenCommand.command, mvnArgs, {
+      cwd: mavenCommand.location || targetFilePath,
```

Comment:
What do you think about making local variable for `cwd`, cos it would be easier later for refactor if we need it?
`const cwd = mavenCommand.location || targetFilePath` and re use it here and on line 119?